changelogs/v0.25.0.md

```diff
@@ -0,0 +1,133 @@
+# v0.25.0 Changes
+
+Initial support for the [Pomerium Zero](https://www.pomerium.com/pomerium-zero/) closed beta is included in this release.
+
+### Breaking
+
+Changes that are expected to cause an incompatibility.
+
+* **config**: remove support for base64 encoded certificates in the [`certificates`](https://www.pomerium.com/docs/reference/certificates#certificates) field. It may only contain file locations. See https://github.com/pomerium/pomerium/pull/4718 by @calebdoxsey for details.
+* **config**: remove [`debug`](https://www.pomerium.com/docs/reference/debug#summary) option, always use json logs by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4857
+
+### New
+* **authenticate**: Refactoring identity authenticators to initiate redirect. For AWS Cognito, please allow the following sign out `https://{AUTHENTICATE_DOMAIN}/.pomerium/signed-out` URL. See more details in https://github.com/pomerium/pomerium/pull/4858 by @calebdoxsey.
```

Comment:
This should actually be `signed_out` with an underscore, right? (see [handlers.go#L91](https://github.com/pomerium/pomerium/blob/b3cb21e13c99f2aa56c80ce313db5a7e4c2c23b8/authenticate/handlers.go#L91))